docs/guides/errors.md

```diff
@@ -0,0 +1,156 @@
+# Errors
+
+All the errors that Prism returns conform to [RFC7807](https://tools.ietf.org/html/rfc7807) - Problem Details for HTTP APIs, which means you'll always get a **JSON object** with the following properties:
+
+* `type` (string)
+* `title` (string)
+* `status` (number)
+* `detail` (string)
+
+This document enumerates all the possible errors returned by Prism and provides some further information on how to solve them.
+
+## Routing errors
+
+This class of errors is returned when Prism is trying to identify the right resource to use to respond to the provided HTTP Request.
+
+### NO_BASE_URL_ERROR
+**Message: Attempted to make a request to a server but neither baseUrl param provided nor servers defined in the spec**
+#### Returned Status Code: `400`
```

Comment:
This doesn’t seem like it’s a header it’s just some more information.

Reply:
Yeah that’s what I meant. This is content which does not need to be a h4 or hAnything.